internal/configs/annotations.go

```diff
@@ -93,29 +89,41 @@ func parseAnnotations(ingEx *IngressEx, baseCfgParams *ConfigParams, isPlus bool
 		if err != nil {
 			glog.Error(err)
 		}
-		cfgParams.HealthCheckEnabled = healthCheckEnabled
+		if isPlus {
```

Comment:
I removed this section in the previous PR, but I think it's safest to put it back for now.

Reply:
I think I'll leave it in, and we might do further refactorings on `annotations.go` sometime in a future PR.